<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="tlValor1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Valor 1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leValor1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="tlValor2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Valor 2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leValor2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>120</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Operações</string>
    </property>
    <widget class="QRadioButton" name="rbAdicao">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Adição</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbSubtracao">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Subtração</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbMutiplicacao">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Multiplicação</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbDivisao">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Divisão</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="tlResultado">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Resultado</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leResultado">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCalcular">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>406</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
